scripts/microbe_stage/organelle.as

```diff
@@ -226,6 +227,9 @@ class Organelle{
 
     //! Cost in mutation points
     int mpCost = 0;
+
+    //! boolean that represent if organelle need presence of nucleus
```

Comment:
This should clarify that this is checked by the editor when attempting to place. And a good thing would also be to use this in the future in autoevo.

Edit and a bit more grammatically pleacing name would be `needsNucleus`